core/src/main/java/gyro/core/command/AbstractConfigCommand.java

```diff
@@ -196,11 +198,23 @@ private void refreshResources(RootScope scope) {
             String name = DiffableInternals.getName(resource);
 
             try {
+                Node credentialNode = DiffableInternals.getScope(resource)
```

Comment:
I'd look into implementing this as a ChangeProcessor. Similar to `ConfiguredFieldsChangeProcessor`.